WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -474,8 +474,9 @@ private extension CardPresentPaymentsOnboardingUseCase {
         return codGateway.enabled
     }
 
-    func isInUndefinedState(account: PaymentGatewayAccount) -> Bool {
-        account.wcpayStatus == .unknown
+    func canCollectCardPresentPayments(account: PaymentGatewayAccount) -> Bool {
```

Comment:
```suggestion
    func accountStatusAllowedForCardPresentPayments(account: PaymentGatewayAccount) -> Bool {
```

Reply:
As mentioned below, changed in 15b918d9f9e786d44943060f23e22b7ed5ff4658